sw/control_sw/config/vla_f_config.yaml

```diff
@@ -2,24 +2,38 @@ fengines:
     chans_per_packet: 32
     pcie64:
         0:
-            feng_ids: [0, 1]
+            # feng_ids defaults to [0, 1], 
+            # feng_ids: [0, 1]
             gbes: ['100.100.102.20']
+            # dests defaults to the keys of xengines['chans']
             source_port: 10000
             dts_lane_map: [8,9,6,7,10,11,1,0,4,5,3,2]
         1:
             feng_ids: [2, 3]
             gbes: ['100.100.102.21']
+            dests: '100.100.103.100-10000' # all feng_ids
             source_port: 10000
             dts_lane_map: [8,9,2,3,10,11,4,5,0,1,6,7]
     pcie65:
         0:
             feng_ids: [4, 5]
             gbes: ['100.100.102.22']
+            dests: [
+                '100.100.102.100-10000', # feng_ids[0]
+                '100.100.103.100-10000' # feng_ids[1]
+            ]
             source_port: 10000
             dts_lane_map: [8,9,10,11,4,5,6,7,0,1,2,3]
         1:
             feng_ids: [6, 7]
             gbes: ['100.100.102.23']
+            dests: [
+                [ # feng_ids[0]
+                    '100.100.102.100-10000',
+                    '100.100.103.100-10000'
```

Comment:
Here we send pcie65_1, feng_id6 to these 2 destinations. The destinations specify which channels they ought to receive.